modules/syslogformat/syslog-format.c

```diff
@@ -954,7 +954,7 @@ _syslog_format_parse_legacy(const MsgFormatOptions *parse_options,
     msg->flags |= LF_UTF8;
 
   log_msg_set_value(msg, LM_V_MESSAGE, (gchar *) src, left);
-
+  log_msg_set_value_to_string(msg, LM_V_MSGFORMAT, "rfc3164");
```

Comment:
A `return TRUE` path has been missed a few lines above.

Reply:
I have fixed this by extracting the function that stores the message itself into a separate function. That should take care of this bug and make it more readable.